draft-private-access-tokens.md

```diff
@@ -1201,7 +1201,7 @@ a single valid public key for signing tokens at a time. The Client does not pres
 of the token public key to the Issuer, but instead expects the Issuer to infer the correct key based
 on the information the Issuer knows, specifically the origin_name itself.
 
-## Issuer and Mediator Ownership
+## Issuer, Mediator and Origin Ownership
```

Comment:
I'm not sure this makes sense to have "origin" under "ownership" here, since we talk only about collusion and not ownership for this case.